kustomize/base/quay.service.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   labels:
     quay-component: quay-app
 spec:
-  type: LoadBalancer
+  type: ClusterIP
```

Comment:
Can this change make a z-stream update fail?

Reply:
Potentially, I'll check and make sure. We already have a utility function which re-creates objects with immutable fields (we use for `Jobs`), so it may automatically do the right thing.